app/views/sandbox/show.html.erb

```diff
@@ -1,16 +1,45 @@
 <% content_for :title, "API sandbox" %>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl"><%= t('page_titles.sandbox') %></h1>
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">Use our sandbox to test Early Career Framework</h1>
+            <p>This is an API test environment which lets you simulate the Early Career Framework service as a training provider.</p>
+            <p>You can:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>sign in as a training provider and confirm the start of an applicant's course</li>
+             <!--    <li>sign in as a training provider and process an applicant</li>
+                <li>test the system’s API capabilities</li> -->
+            </ul>
+            <p>You can give us feedback at any time to help us improve the service</p>
+            <h2 class="govuk-heading-m">Get started</h2>
+            <p> The ECF service allows you to integrate your software with ours and test it.</p>
```

Comment:
Extra space after the opening <p>

Reply:
Not really a big deal for html parsing, but for someone looking at the source, it's obvious